ext/date/date_core.c

```diff
@@ -3388,37 +3416,37 @@ date_s_ordinal(int argc, VALUE *argv, VALUE klass)
     return ret;
 }
 
+static VALUE
+date_s_civil(int argc, VALUE *argv, VALUE klass)
```

Comment:
I think we should have some documentation saying this is the same as `Date.new`.

Reply:
I've replaced the long doc with just a phrase.  (Are they really identical?)